prow/plugins/blunderbuss/blunderbuss.go

```diff
@@ -61,6 +61,9 @@ func helpProvider(config *plugins.Configuration, enabledRepos []string) (*plugin
 }
 
 type ownersClient interface {
```

Comment:
Seems like you should provide a different implementation of the `ownersClient` that returns approvers instead of duplicating methods

Reply:
:+1: I'll experiment with that approach.

Reply:
@stevekuznetsov RFAL.